uint8 service
uint8 signal_strength
uint8 antenna
